templates/search/featured_search.html

```diff
@@ -11,49 +11,28 @@
     <p class="search-results__explainer--desktop">This page displays a summary of catalogue and non-catalogue content that matches your search.</p>
     <p class="search-results__explainer--mobile">Catalogue and non-catalogue content that matches your search.</p>
     <div class="featured-search__results">
-        {% with heading='Records from The National Archives' group="group:tna" %}
+        {% with results=result_groups.tna%}
```

Comment:
Nice use of `with` here! When 'variabalizing' things specifically for an include to pick up, it can be cleaner to do:
```
{% include 'template_name' with results=results_groups.tna %}
```

Reply:
Thanks @ababic - I've changed them to reflect this now